broker/src/main/java/io/zeebe/broker/system/partitions/impl/steps/StateControllerPartitionStep.java

```diff
@@ -28,7 +28,7 @@
     final var stateController =
         new StateControllerImpl(
             context.getPartitionId(),
-            DefaultZeebeDbFactory.defaultFactory(databaseCfg.getColumnFamilyOptions()),
+            DefaultZeebeDbFactory.defaultFactory(databaseCfg.createRocksDbConfiguration()),
```

Comment:
I'm now wondering whether we havent used the configs before ? :thinking:

Reply:
I think not because I ran into this problem initially (which is why I added the log statement at the top haha) :see_no_evil:

Reply:
lol nice :D